application/controllers/ReportController.php

```diff
@@ -49,14 +51,29 @@ public function indexAction()
     {
         $this->addTitleTab($this->report->getName());
 
+        $this->controls->getAttributes()->add('class', 'default-layout');
         $this->addControl($this->assembleActions());
 
+        $contentId = $this->content->getAttributes()->get('id')->getValue();
+        $this->sendExtraUpdates([
+            $contentId => Url::fromPath('reporting/report/content', ['id' => $this->report->getId()])
+        ]);
+
+        // Will be replaced once the report content is rendered
+        $this->addContent(new HtmlElement('p'));
```

Comment:
Consider to use a element without a "meaning". I.e. `HtmlString::create('&nbsp;')` or `div`.